### DISCUSSION TITLE
Application  not compiling after saving changes

### DISCUSSION THREAD

**Discussion by 19jmrs** [vercel/next.js#61585] [Help]
### Summary

Every time I run the command **npm run dev** the application is compiled correctly initially, but if I perform some changes it does not recompile to reflect the new changes. This behavior started to happen for no apparent reason as it was working fine before.

I already verified if all the imports are with the correct case sensitive and they all are.

Getting out of ideas of what else I could try.

Suggestions are appreciated.

**Comment by Skecsup** [comment]
I have the same issue, have you found the solution?
